agents/a2a_mcp_integration.py

```
@@ -297,345 +324,494 @@ async def _send_mcp_pipe(self, message: A2AMCPMessage) -> Dict[str, Any]:
+            logger.error(f"Tool execution error: {e}")
+            return {"status": "error", "error": str(e), "tool": tool_name}
+
+    async def _execute_tool_direct(
+        self, tool_name: str, params: Dict[str, Any]
+    ) -> Dict[str, Any]:
+        """Direct tool execution when MCP server is unavailable"""
+        if tool_name == "process_data":
+            from protocols.data_processor import task as process_data_task
+
+            return process_data_task()
+        elif tool_name == "quantum_optimize":
```

Ruff (0.11.9)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)